prime-angular-frontend/src/app/modules/adjudication/pages/site-overview/site-overview.component.ts

```diff
@@ -81,6 +89,31 @@ export class SiteOverviewComponent implements OnInit {
     }
   }
 
+  public saveVendor(): void {
+    const existingVendor = this.siteVendors.find((vendor: VendorConfig) => vendor.code === this.site.siteVendors[0].vendorCode).name;
+    const vendor = this.vendors.value;
+
+    if (this.form.get('vendors').valid && existingVendor !== vendor.name) {
```

Comment:
Should use property instead of `this.form.get('vendors')`

Reply:
it is updated.